chart/dockerfiles/pgbouncer-exporter/Dockerfile

```diff
@@ -28,13 +28,19 @@ LABEL org.apache.airflow.airflow-pgbouncer-exporter.version="${AIRFLOW_PGBOUNCER
 LABEL org.apache.airflow.commit-sha="${COMMIT_SHA}"
 LABEL maintainer="Apache Airflow Community <dev@airflow.apache.org>"
 
+# Add a non root user
+RUN addgroup -S pgbouncer_exporter \
```

Comment:
What do you think about support arbitrary user ids? https://docs.openshift.com/container-platform/4.9/openshift_images/create-images.html#images-create-guide-openshift_create-images

Reply:
we are defining userid's in securityContext: , if the securityContext is  left black or id's are in range of openshift we wont have any issues.

Reply:
How about using user "nobody" (Like in the pgbouncer itself). I think this case is not the same as airflow - in airflow often people want to use different user or use the "arbitrary user id" to avoid some potential "user' clashes between containers running on the same machine and they evn might want to choose a specific ID because they can have volumes mounted with specific users etc. So it makes sense to support the "arbitrary user id". We also created a home directory there because airflow workers might spawn pretty much arbitrary operators (thus any external libraries/binaries) and many of those extarnal binaries/librraries might rely on the ${HOME} directory set and present (and writeable) - for example when they do any caching, they often do that in the ${HOME} dir.

But `pgbouncer_exporter` is really simple, "standalone" binary tool that is rather simple - it connects to pgbouncer, queries it for stats and exposes to prometheus via non-priviledged port (9127 which is > 1024) - so there is no need for any special priviledges, also it is unlikely it needs HOME directory (we can check that likely).

This is nobody user:

```
nobody:x:65534:65534:nobody:/nonexistent:/usr/sbin/nologin
```

WDYT?

Reply:
If we use nobody, it makes a bit simpler.

Reply:
yeah both approaches are right and USER nobody option is simpler this service just exposes metrics on http endpoint. Thanks @potiuk

Reply:
Can you make that change @pgvishnuram so we can merge this PR